tests/ccgbugs2/tcodegen.nim

```diff
@@ -0,0 +1,27 @@
+discard """
```

Comment:
Why is the file called `tcodegen`?

Reply:
The original issue is labelled with codegen. Naming it as "tcodegen" means all the codegen bugs which now compile can fall into this file.